### PR TITLE
Update clients ubuntu eins (Uyuni 2022.09)

### PR DESCRIPTION
**Pull request by eins**
# Description

Short summary of why you created this PR

Following the issue [5804](https://github.com/uyuni-project/uyuni/issues/5804) 

I update docs in order to have the correct file base channel ubuntu-2204-pool-amd64-uyuni

# Target branches

Which documentation version does this PR apply to?

- [ X ] Master (Default)

### CODE REVIEW

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 1]
```diff
@@ -69,8 +69,8 @@ The channels you need for this procedure are:
 
 | OS Version | Base Channel | Main Channel | Updates Channel | Security Channel | Client Channel
 
-| {ubuntu} 20.04 | ubuntu-2004-amd64-main for amd64 | ubuntu-2004-amd64-main-uyuni | ubuntu-2004-amd64-main-updates-uyuni | ubuntu-2004-amd64-main-security-uyuni | ubuntu-2004-amd64-uyuni-client
-| {ubuntu} 22.04 | ubuntu-2204-amd64-main for amd64 | ubuntu-2204-amd64-main-uyuni | ubuntu-2204-amd64-main-updates-uyuni | ubuntu-2204-amd64-main-security-uyuni | ubuntu-2204-amd64-uyuni-client
+| {ubuntu} 20.04 | ubuntu-2004-pool-amd64-uyuni for amd64 | ubuntu-2004-amd64-main-uyuni | ubuntu-2004-amd64-main-updates-uyuni | ubuntu-2004-amd64-main-security-uyuni | ubuntu-2004-amd64-uyuni-client
+| {ubuntu} 22.04 | ubuntu-2204-pool-amd64-uyuni for amd64 | ubuntu-2204-amd64-main-uyuni | ubuntu-2204-amd64-main-updates-uyuni | ubuntu-2204-amd64-main-security-uyuni | ubuntu-2204-amd64-uyuni-client
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc:73]: I think we should additionally remove the descriptive ` for amd64` in that table cells.

Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-ubuntu.adoc:73]:
```suggestion
| {ubuntu} 20.04 | ubuntu-2004-pool-amd64-uyuni | ubuntu-2004-amd64-main-uyuni | ubuntu-2004-amd64-main-updates-uyuni | ubuntu-2004-amd64-main-security-uyuni | ubuntu-2004-amd64-uyuni-client
| {ubuntu} 22.04 | ubuntu-2204-pool-amd64-uyuni | ubuntu-2204-amd64-main-uyuni | ubuntu-2204-amd64-main-updates-uyuni | ubuntu-2204-amd64-main-security-uyuni | ubuntu-2204-amd64-uyuni-client
```

**Review by juliogonzalez — APPROVED** [review]

**Review by eins — COMMENTED** [review]
@keichwa   I removed the descriptive for amd64 in that table cells.

**Review by keichwa — APPROVED** [review]
thank you, looks good now.
I added a changelog entry.

If nobody vetoes I'll merge it later today.

**Pull request merged**